app/elastic_search_indexes/cache/heading_index.rb

```diff
@@ -1,18 +1,23 @@
 module Cache
   class HeadingIndex < ::Cache::CacheIndex
+    def dataset
+      TimeMachine.now do
+        super.actual
+      end
+    end
+
     def eager_load_graph
       [
         {
-          commodities: [
-            :goods_nomenclature_descriptions,
-            :goods_nomenclature_indents,
-            { heading: [:commodities] }, # TODO: We should be able to replace loading the heading and its commodities with the materialized path concept and avoid eager loading for the goods nomenclature mapper
```

Comment:
Fast declarable does not ask for the heading and its commodities for the goods nomenclature mapper anymore so this is now redundant